x-pack/plugins/cases/public/common/use_cases_toast.tsx

```diff
@@ -34,6 +34,20 @@ const EuiTextStyled = styled(EuiText)`
   `}
 `;
 
+function getAlertsCount(attachments: CaseAttachments): number {
+  let alertsCount = 0;
+  for (const attachment of attachments) {
+    if (attachment.type === CommentType.alert) {
+      if (Array.isArray(attachment.alertId) && attachment.alertId.length > 1) {
```

Comment:
Nit: this might be easier to read, but 100% a personal preference thing 😅

```
if (Array.isArray(attachment.alertId)) {
	alertsCount += attachment.alertId.length;
} else {
  alertsCount += 1
}
```